pkgs/development/python-modules/meraki/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3
+, aiohttp
+, requests
+, setuptools
+, typer
+}:
+
+buildPythonPackage rec {
+  pname = "meraki";
+  version = "1.30.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-s26xGwWSWB+qpOTUe8IYo53ywYOaaUWjDznFqpmRlak=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    requests
+    setuptools
+  ] ++ typer.optional-dependencies.all;
```

Comment:
The only dependencies I see actually referenced as required at runtime by `setup.py` are `aiohttp` and `requests`. Does this really need `setuptools` in `propagatedBuildInputs` and does it need the `typer` deps at all? (I don't see them referenced even in the development `requirements.txt` file in their repo)

Reply:
I don't think so, it certainly just built without them. I think I copied this block from another package as a starter and just never removed those. I'll push a fix now.